README.md

```diff
@@ -68,7 +68,9 @@ You can run the following command to test omni with that repository:
 omni clone https://github.com/omnicli/omni-example-go.git
 ```
 
-## Configuration
+## Setting up omni for your repo
```

Comment:
At least to me: this section being called "Configuration" was confusing because I expected it meant configuring omni itself, not how omni will act in certain repos